app/src/main/res/xml/preferences.xml

```diff
@@ -109,6 +109,14 @@
             android:key="prefHardwareForwardButtonSkips"
             android:summary="@string/pref_hardwareForwardButtonSkips_sum"
             android:title="@string/pref_hardwareForwardButtonSkips_title"/>
+        <Preference
```

Comment:
Ahh. Now I see what you mean.  (the github comment viewer is not showing the best context for these code changes).

I patterned this off of the playback speed preference.    Let me take a look at using the MaterialsListPreference....